opencti-platform/opencti-graphql/src/database/data-initialization.js

```diff
@@ -16,7 +16,7 @@ import { TAXIIAPI } from '../domain/user';
 import { KNOWLEDGE_COLLABORATION, KNOWLEDGE_DELETE, KNOWLEDGE_MANAGE_AUTH_MEMBERS, KNOWLEDGE_UPDATE } from '../schema/general';
 
 // region Platform capabilities definition
-const KNOWLEDGE_CAPABILITY = 'KNOWLEDGE';
+export const KNOWLEDGE_CAPABILITY = 'KNOWLEDGE';
```

Comment:
I think it would be better to move this constant in utils/access.ts, then import it here (and in log.ts) , instead of export this constant from this file.